Client/Frontend/Theme/Theme.swift

```diff
@@ -53,14 +53,18 @@ class ActionMenuColor {
 }
 
 class URLBarColor {
-    var border: UIColor { return UIColor.Photon.Grey50 }
+    var border: UIColor { return UIColor.Photon.Grey90A10 }
```

Comment:
This is the same as before. The grey 50 had an alpha applied that made it equivalent to this color.